charger/delta.go

```diff
@@ -162,6 +173,27 @@ func (wb *Delta) Status() (api.ChargeStatus, error) {
 	}
 }
 
+var _ api.StatusReasoner = (*Delta)(nil)
+
+// statusReason implements the api.StatusReasoner interface
```

Comment:
Das könnte man dekorieren wenn nicht immer verfügbar.

Reply:
Ja, könnte man natürlich.
Aber ich erinnere mich an Diskussionen über die dadurch aufgebähte Binarygröße?

Reply:
Nur in Kombinatorik- hier unkritisch.

Reply:
Ok, dann bau ich das noch um.

Reply:
#15983